addons/music/fnc_searchMusic.sqf

```diff
@@ -1,78 +0,0 @@
-/* ----------------------------------------------------------------------------
```

Comment:
What happened to this? did you remove it completly? I kinda liked the feature. But nothing many people would use anyway.